.github/workflows/ci.yml

```diff
@@ -0,0 +1,140 @@
+name: Alfresco Transform Core CI
+
+on:
+  push:
+    branches:
+      - feature/**
```

Comment:
`master`, `SP/**` and `HF/**` branches are missing and I'm not sure if we need` ATS-**`, `ACS-**` and `MNT-** `here.